model/sheet.cc

```diff
@@ -35,11 +35,13 @@ ClickType::Type Sheet::clickType(float x, float y)
 {
   double dx=x-this->x(), dy=y-this->y();
   auto z=zoomFactor();
-  double w=0.5*m_width*z, h=0.5*m_height*z;
-  // check if (x,y) is within portradius of the 4 corners
-  if (fabs(fabs(dx)-w) < portRadiusMult*z &&
-      fabs(fabs(dy)-h) < portRadiusMult*z &&
-      fabs(hypot(dx,dy)-hypot(w,h)) < portRadiusMult*z)
+  //double w=0.5*m_width*z, h=0.5*m_height*z;
+  //// check if (x,y) is within portradius of the 4 corners
+  //if (fabs(fabs(dx)-w) < portRadiusMult*z &&
+  //    fabs(fabs(dy)-h) < portRadiusMult*z &&
+  //    fabs(hypot(dx,dy)-hypot(w,h)) < portRadiusMult*z)
```

Comment:
Remove dead code

Reply:
Sorry, I just missed the chance to make this change.. I was working on your suggestions for ticket 1203 and had to commit all my changes before checking out this branch and removing the dead code.

Reply:
No worries - I fixed this up when merging feature #94